PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/common/AzureDialog.java

```diff
@@ -65,11 +65,17 @@ public void close() {
     @Override
     protected List<ValidationInfo> doValidateAll() {
         final List<AzureValidationInfo> infos = this.getForm().validateData();
-        this.setOKActionEnabled(infos.stream().noneMatch(i -> i == AzureValidationInfo.PENDING || i.getType() == AzureValidationInfo.Type.ERROR));
-        return infos.stream()
-                    .filter(i -> i != AzureValidationInfo.PENDING && i != AzureValidationInfo.OK)
+        this.setOKActionEnabled(infos.stream().noneMatch(
+                i -> i == AzureValidationInfo.PENDING || i.getType() == AzureValidationInfo.Type.ERROR || AzureValidationInfo.UNINITIALIZED.equals(i)));
+        List<ValidationInfo> resultList = infos.stream()
+                    .filter(i -> i != AzureValidationInfo.PENDING && i != AzureValidationInfo.OK && !AzureValidationInfo.UNINITIALIZED.equals(i))
                     .map(AzureDialog::toIntellijValidationInfo)
                     .collect(Collectors.toList());
+        // this is in order to let ok action disable if only there is any uninitialized filed.
+        if (infos.stream().filter(e -> AzureValidationInfo.UNINITIALIZED.equals(e)).count() > 0L) {
+            setErrorInfoAll(resultList);
```

Comment:
shall we make UNINITALIZED as an error validation info?

Reply:
no. we just regard it as valid state, not a error one.